policy/modules/services/xserver.te

```diff
@@ -33,6 +33,14 @@ gen_require(`
 ## </desc>
 gen_tunable(allow_write_xshm, false)
 
+## <desc>
+## <p>
+## Allows clients to write to the X server tmpfs
+## files.
+## </p>
+## </desc>
+gen_tunable(allow_write_xserver_tmpfs, false)
```

Comment:
On second thought, as with the other tunable you recently added, I'd like to make this name start with `xserver`. How about `xserver_client_rw_tmpfs`?

Reply:
Then allow_write_xshm also needs to be renamed. I'll do that.

Reply:
Also, the "rw" wording would be misleading, as the read permission is always granted...

Reply:
> Then allow_write_xshm also needs to be renamed. I'll do that.

The problem is that breaks backward compatibility.

> Also, the "rw" wording would be misleading, as the read permission is always granted...

Fair enough.

Reply:
The `allow_write_xshm` needs to be restored.

Reply:
I have now restored the original naming, so it should be ready to merge.